doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,255 @@
                     # ...
                 )
                 # ...
 
                 loss = optimize()
 
                 metrics = {"loss": loss}
-                # Only report the first worker results to wandb
-                if context.get_world_rank() == 0:
+                # Only report the first worker results to wandb to avoid dup
+                if train.get_context().get_world_rank() == 0:
                     wandb.log(metrics)
 
-                # Also report to Ray Train. Note that this _must_ happen for all workers.
-                train.report(metrics)
+    .. tab:: file based MLflow
 
-    .. tab-item:: PyTorch Lightning
+        .. code-block:: python
+            
+            from ray import train
+            import mlflow
 
-        You can keep using the Lightning's native Logger integrations in the worker 
-        function. They should work out-of-the box with Ray TorchTrainer.
+            # assuming passing a save dir through config
+            def train_func(config):
+                save_dir = config["save_dir"]
+                mlflow.start_run(tracking_uri=f"file:{save_dir}")
 
-        Example:
+                # ...
 
-        .. code-block:: python
-            
-            import pytorch_lightning as pl
-            from pytorch_lightning.loggers.wandb import WandbLogger
-            from pytorch_lightning.loggers.comet import CometLogger
-            from pytorch_lightning.loggers.mlflow import MLFlowLogger
-
-            def train_func_per_worker():
-                ...
-
-                wandb_logger = WandbLogger(
-                    name="demo-run", 
-                    project="demo-project", 
-                    id="unique_id",  
-                    offline=offline
-                )
-                
-                comet_logger = CometLogger(
-                    api_key=YOUR_COMET_API_KEY,
-                    experiment_name="demo-experiment",
-                    project_name="demo-project,
-                    offline=offline,
-                )
-                
-                mlflow_logger = MLFlowLogger(
-                    run_name=name,
-                    experiment_name=project_name,
-                    tracking_uri=f"file:{save_dir}/mlflow",
-                )
-                
-                trainer = pl.Trainer(
-                    # ...,
-                    logger=[wandb_logger, comet_logger, mlflow_logger],
-                )
+                loss = optimize()
 
-        .. tip::
+                metrics = {"loss": loss}
+                # Only report the first worker results to mlflow to avoid dup
+                if train.get_context().get_world_rank() == 0:
+                    mlflow.log_metrics(metrics)
 
-            Always make sure to set necessary credentials on each worker.
-            Ray Train will not automatically port environment varaibles 
-            on the head node to workers.
+    .. tab:: MLflow externally hosted by databricks
```

Comment:
Is it necessary to show this? Wondering how much value this provides, or if the user can just easily adapt from the previous example.